src/leap/mail/imap/service/imap-server.tac

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # imap-server.tac
 # Copyright (C) 2013,2014 LEAP
```

Comment:
fixed, thanks. I think with the new bitmask.core.bitmaskd we can deprecate this .tac, its use was mainly to be able to instantiate a local imap server quickly.